__tests__/components/sections/feedbacks/Form.test.tsx

```diff
@@ -0,0 +1,29 @@
+import Form from "@/components/sections/feedbacks/Form"
+import "@testing-library/jest-dom"
+import { fireEvent, render, screen } from "@testing-library/react"
+import ReduxProvider from "../../../../__mocks__/provider/ReduxProvider"
+jest.mock("next/router", () => ({
+    useRouter: () => ({
+        push: jest.fn(),
+        isFallback: false,
+    }),
+}));
+const handleSave = jest.fn()
+const renderForm = () => {
+    render(<ReduxProvider><Form onSave={handleSave} /></ReduxProvider>)
+
+}
+describe('FeedbackForm', () => {
+    it('should render the feedback form', () => {
+        renderForm()
+        const form = screen.getByTestId('feedback-form')
+        expect(form).toBeInTheDocument()
+    })
+
+    it('should not make modifications to the text input', () => {
```

Comment:
Should have the value passed in the input

Reply:
Since I am testing the whole form, there is no way to add the value in the input, below is the line i uses

```bash
    fireEvent.change(inputText, { target: { value: 'Test input' } })
```